packages/schema/lib/schemas/RequestSchema.js

```diff
@@ -80,6 +80,12 @@ module.exports = makeSchema(
         description:
           'A function to customize how to serialize a value for curlies `{{var}}` in the request object. By default, when this is unspecified, the request client only replaces curlies where variables are strings, and would throw an error for non-strings. The function should accepts a single argument as the value to be serialized and return the string representation of the argument.',
         $ref: FunctionSchema.id
+      },
+      skipThrowRequest: {
+        description:
+          'Should throwForStatus be calld before returning the response?',
```

Comment:
Ditto, we can be more specific by saying:

> If true, throw an exception for response status >= 400 automatically before returning the response. Defaults to `false`.

Reply:
Done, but using `> 300` for now.